### PR TITLE
feat: Using web3 migrations

### PR DESCRIPTION
**Pull request by classicalliu**
Related PR:
- https://github.com/nervosnetwork/godwoken-web3/pull/346

### CODE REVIEW

**Review by Flouse — APPROVED** [review]

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -107,24 +106,17 @@ services:
...
+  web3-migrations:
+    image: ghcr.io/nervosnetwork/godwoken-web3-prebuilds:refactor-db-01b5da5
     volumes: 
       - ./layer2/config/web3-config.env:/godwoken-web3/packages/api-server/.env
-      - ./web3/entrypoint.sh:/var/lib/web3/entrypoint.sh
-    entrypoint: /var/lib/web3/entrypoint.sh
-    ports:
-      - 8024:8024
+      - ./web3/migrations_entrypoint.sh:/var/lib/web3/migrations_entrypoint.sh
+    entrypoint: /var/lib/web3/migrations_entrypoint.sh
```
Comment (keroro520) [inline docker/docker-compose.yml:114]: Would it be possible that do the migration job inside [`web3/entrypoint.sh`](https://github.com/RetricSu/godwoken-kicker/blob/compatibility-changes/docker/web3/entrypoint.sh).

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: docker/docker-compose.yml (hunk 1 above)
Comment (keroro520) [reply docker/docker-compose.yml:114]:
```
cd /godwoken-web3

// Question: Should we check whether the migration job should start?
yarn knex migrate:latest

yarn run start
```

Comment (classicalliu) [reply docker/docker-compose.yml:114]: web3-indexer also need to start after migrations completed, how web3-indexer get the status or start web3-indexer after web3 server already started ?

Comment (keroro520) [reply docker/docker-compose.yml:114]: webs-indexer will wait until web3 becomes healthy:
https://github.com/RetricSu/godwoken-kicker/blob/0880fdf244daf9bacb8617cc8d4e0fce630ce1e6/docker/docker-compose.yml#L155-L157

Comment (classicalliu) [reply docker/docker-compose.yml:114]: ok, I see

**Review by keroro520 — APPROVED** [review]

**Pull request merged**